### PR TITLE
[For v0.18]Sync TechDraw Toolbars/Menu and WhatsThis/Wiki

### PR DESCRIPTION
**Pull request by WandererFan**
This PR synchronizes the TechDraw Toolbars and the TechDraw Menu entries.  It also changes the "WhatsThis" test of TechDraw commands to match the corresponding Wiki page.  Please merge. 

Thanks,
wf

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [ ] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.

---

### CODE REVIEW

File: src/Mod/TechDraw/Gui/CommandCreateDims.cpp [hunk 1]
```diff
@@ -234,9 +236,9 @@ CmdTechDrawNewRadiusDimension::CmdTechDrawNewRadiusDimension()
...
     sAppModule      = "TechDraw";
     sGroup          = QT_TR_NOOP("TechDraw");
-    sMenuText       = QT_TR_NOOP("Insert a new radius dimension into the drawing");
-    sToolTipText    = QT_TR_NOOP("Insert a new radius dimension feature for the selected view");
-    sWhatsThis      = "TechDraw_NewRadiusDimension";
+    sMenuText       = QT_TR_NOOP("Insert a new radius dimension");
+    sToolTipText    = QT_TR_NOOP("Insert a new radius dimension");
+    sWhatsThis      = "TechDraw_Dimension_Radius";
     sStatusTip      = sToolTipText;
     sPixmap         = "TechDraw_Dimension_Radius";
```
Comment (luzpaz) [inline src/Mod/TechDraw/Gui/CommandCreateDims.cpp]: @WandererFan any idea why this sPixmap is different than the ones previous (which are shown as a path) ?

**Comment by wwmayer** [comment]
Note, that this PR should be merged in v0.18 because it changes a lot of translation strings which would break again all the .ts files.

**Comment by WandererFan** [comment]
@luzpaz: History?  The TD icon directory has always had an "actions" subdirectory and the Dimension icons have always been in the main "icons" directory with the tree symbols.  It wasn't broken, so it never got "fixed".

**Comment by luzpaz** [comment]
@wandererfan makes sense

**Comment by normandc** [comment]
@wwmayer can't at least commit 898cafd be merged? It won't affect translations, and without it the Qt Assistant Help file will be completely broken where TechDraw is concerned.

**Comment by luzpaz** [comment]
>  without it the Qt Assistant Help file will be completely broken where TechDraw is concerned.   

In other words, 0.17 will be shipped with broken TD help functionality

**Comment by normandc** [comment]
> In other words, 0.17 will be shipped with broken TD help functionality

Yes. BTW ulrich1a mentioned on the forum we have the same issue with Sketcher constraint command pages and Part Measure tools. https://forum.freecadweb.org/viewtopic.php?f=21&t=27265#p218564

**Comment by yorikvanhavre** [comment]
merged

**Comment by wwmayer** [comment]
@normandc actually commit 898cafd could be merged to the 0.17 branch but it looks like it doesn't make a difference because the user documentation (the qch file) doesn't contain the Techdraw pages.

**Comment by wwmayer** [comment]
OK, I had to unregister and re-register the qch file in QtAssistant to update the table of contents. Now it finds the TechDraw pages. What's also odd is that the above steps modifies the qch file.

**Pull request closed without merging**